spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
Awesome!

Can you add a new `context` at the top-level (same level as describe) for making sure this works in non-js enabled tests? Like this:

```ruby
context 'rack-test without javascript', type: :feature do
  # ... duplicate the `it` test here
end
```